packages/core/src/Toast/toastCreators.tsx

```diff
@@ -219,13 +233,13 @@ export const createInfoToast: SimpleToastCreator = ({
     </ToastLabel>
   )
   const icon = (
-    <InfoIconColor>
+    <ToastIconWrapper>
```

Comment:
iconType={ToastIconType.INFO} ??

Reply:
It's the default value. But I guess I could be explicit if you'd like.

Reply:
Ok. I didn't notice that.

Reply:
I fixed it. :+1: